services/QuillLMS/app/models/sales_form_submission.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class SalesFormSubmission < ApplicationRecord
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :email, presence: true
+  validates :phone_number, presence: true
+  validates :zipcode, presence: true
+  validates :collection_type, presence: true
+  validates :school_name, presence: true
+  validates :district_name, presence: true
+  validates :school_premium_count_estimate, presence: true
```

Comment:
For the three integer fields, what about "validates [FIELD_NAME], numericality: { greater_than_or_equal_to: 0 }`

Reply:
thanks for all of the validation suggestions! I'll add those in, as well as adding default values for the numerical fields